spec/javascripts/VariableSpec.js

```diff
@@ -330,6 +330,67 @@ describe("Variable", function() {
 
 	});
 
+	describe("meta", function() {
+		var defaultVariableConfig = {
+			key: "value",
+			opts: {
+				title: "Red title",
+				is_expand: true,
+				meta: {
+					is_expose: false
+				}
+			}
+		};
+
+		it("should get meta from user variable config", function() {
+			var variable = new Variable({
+				key: "value",
+				opts: {
+					title: "Blue title",
+					is_expand: true,
+					meta: {
+						is_expose: false
+					}
+				}
+			}, defaultVariableConfig);
+
+			expect(variable.meta()).toEqual({
+				is_expose: false
+			});
+		});
+
+		it("should get meta from default variable config if user variable config is not defined", function() {
+			var variable = new Variable(undefined, defaultVariableConfig);
+
+			expect(variable.meta()).toEqual({
+				is_expose: false
+			});
+		});
+
+
+		it("should override meta", function() {
+			var variable = new Variable({
+				key: "value",
+				opts: {
+					title: "Blue title",
+					is_expand: true,
+					meta: {
+						is_expose: false
+					}
+				}
+			}, defaultVariableConfig);
+
+			var meta = {
+				is_expose: true,
+				is_updated: false
+			};
+
+			expect(variable.meta(meta)).toEqual(meta);
```

Comment:
`variable.meta(meta)` is a setter. Although in our case it also returns the value it sets, it is not a required behaviour from getter-setter methods' setter mode (sometimes setters return true/false, indicating success/fail) and even in our case it might change in the future. So we should not expect the method to return the meta when used as a setter. So first use this separately as a setter, and in the expect, only use it as a getter.